enet/PSPBUILD

```diff
@@ -0,0 +1,34 @@
+pkgname=enet
+pkgver=1.3.17
+pkgrel=1
+pkgdesc="ENet reliable UDP networking library"
+arch=('mips')
+url="http://enet.bespin.org/"
+license=()
+depends=()
+makedepends=()
+optdepends=()
+source=("git+https://github.com/lsalzman/enet.git#commit=74cea7abf52ddd355146aeb0a4077d2b95368122")
```

Comment:
This commit is not version 1.3.17. Please use the tar.gz for the tag or make the version include the commit hash.

Reply:
Done